src/index.js

```diff
@@ -0,0 +1,35 @@
+import render from './render';
+import sel from './helpers';
+import './css/style.css';
+import './css/form.css';
+
+const api = '6ca8aaad293b0b0a3ad93f29e82425f8';
+
+const fetchData = async (url) => {
+  const response = await fetch(url, { mode: 'cors' });
+  if (response.status === 401) { return console.error('BAD REQUEST | Access not allowed'); }
+  if (response.status === 404) { return console.error('BAD REQUEST | City not found'); }
+  const data = await response.json();
```

Comment:
- We require in this project to apply the concept of object restructuring. This line is a place where we can apply the concept.
```const {name, main......} = await response.json();```

Reply:
But destructuring happen immediately even with the `await`, and my variable doesn`t have the same names.

Reply:
But destructuring happen immediately even with the `await`, and my variable doesn't have the same names.

Reply:
But destructuring run before `await` happens, and my variable doesn't have the same names.

Reply:
But destructuring run before `await` happens, and my variables don't have the same names.

Reply:
But destructuring run before `await` happens, and my variables doesn't have the same names.

Reply:
But destructuring run before `await` happens, and my variables doesn't has the same names.